common/character_traits/dvg_personality_traits.txt

```diff
@@ -0,0 +1,32 @@
+﻿### Personality
+
+very_persistent = {
```

Comment:
Yes, to give max health.

Reply:
Why is it needed?  I personally don't see it significant to have "immortality"(cus rest is the same) trait for one character in the game

Reply:
and can it be too railroaded? I think in the base game death of William IV is not scripted (cant check rn so I'm not sure) and the game title is around his successor what cand be more important than that

Reply:
Because right now there's no consistent way to have a character die at a certain point. Until Pdox opens up more of the character defines to modding, this is the best way to go right now. Also the death of William IV is scripted in the base game.

Reply:
Now they can, but can be limited using possible. I will remove this trait for now. It's not needed with the other solution for now, although not perfect. 
But I will continue looking for a more general method to make characters immortal. I have a few ideas but need some more time to try them out.